From 0ce5905d4cd5392ae9f7cd12ac52bf39d056a9cb Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Tue, 23 Feb 2016 17:04:13 +0800
Subject: [PATCH 0955/5242] MLK-12433: arm: dts: imx: disable tempmon device
 when cpufreq is disabled

commit  2b78e693ffaec7c8640f7b1d7c5969ab072bef2b from
https://source.codeaurora.org/external/imx/linux-imx.git

In the lpddr3-arm2-m4 dts, the I2C1 is disabled, so PMIC is disabled,
the cpufreq is not support. As thermal driver is depended on cpufreq
driver, if cpufreq is not support, the tempmon device can be disabled.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-m4.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-m4.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-m4.dts
index e06a53f..d5579f2 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-m4.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-m4.dts
@@ -63,6 +63,10 @@
 	status = "disabled";
 };
 
+&tempmon {
+	status = "disabled";
+};
+
 &uart2 {
 	status = "disabled";
 };
-- 
1.7.9.5

